content/en/blog/_posts/2024-09-30-gateway-api-v1.2/index.md

```diff
@@ -0,0 +1,418 @@
+---
+layout: blog
+title: "Gateway API v1.2: Protocols, Labels, and Timeouts, oh my!"
+date: 2024-05-09T09:00:00-08:00
+slug: gateway-api-v1-1
+author: Gateway API Contributors
+---
+
+![Gateway API logo](gateway-api-logo.svg)
+
+Kubernetes SIG Network is delighted to announce the v1.2 release of [Gateway
+API](https://gateway-api.sigs.k8s.io/)! This release brings a number of new
+features to the _Standard Channel_ (Gateway API's GA release channel),
+introduces some new experimental features, and inaugurates our new release
+process -- but it also brings two breaking changes that you'll want to be
+careful of.
+
+## BREAKING CHANGES
+
+### GRPCRoute and ReferenceGrant `v1alpha2` Removal
+
+Now that the `v1` versions of GRPCRoute and ReferenceGrant have graduated to
+Standard, the old `v1alpha2` versions have been removed from both the Standard
+and Experimental channels, in order to ease the maintenance burden that
+perpetually supporting the old versions would place on the Gateway API
+community.
+
+Before upgrading to Gateway API v1.2, you'll want to confirm that any
+implementations of Gateway API have been upgraded to support the v1 API
+version of these resources instead of the v1alpha2 API version. Note that even
+if you've been using v1 in your YAML manifests, a controller may still be
+using v1alpha2 which would cause it to fail during this upgrade. Additionally,
+Kubernetes itself goes to some effort to stop you from removing a CRD version
+that it thinks you're using: check out [the release notes] for more
+information about what you need to do to safely upgrade.
+
+[the release notes]: https://github.com/kubernetes-sigs/gateway-api/releases/tag/v1.2.0
```

Comment:
```suggestion
For more information, see the [release notes](https://github.com/kubernetes-sigs/gateway-api/releases/tag/v1.2.0)
```

Reply:
Yeah, looks like @sftim is particularly noting that the article probably shouldn't be in the link. Fixed that.